filebeat/docs/include/running-modules.asciidoc

```diff
@@ -2,54 +2,119 @@
 [id="running-{modulename}-modules"]
 === Set up and run the module
 
-IMPORTANT: If you’ve secured Elasticsearch and Kibana, you need to configure the
-`username` and `password` options in the Elasticsearch output before setting up
+// REVIEWERS: Should we include docker commands here, too?
```

Comment:
I don't think we should, as it is not straightforward in the docker world to follow these interactive steps